typescript/infra/config/environments/mainnet3/tokenPrices.json

```diff
@@ -14,5 +14,6 @@
   "polygonzkevm": "2914.61",
   "inevm": "32.77",
   "viction": "0.750231",
+  "neutron": "0.750231",
```

Comment:
```suggestion
  "neutron": "1.76",
```

Reply:
from https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3327 when I ran the print token prices script